samples/authenticator/src/main/java/org/forgerock/authenticator/sample/view/activity/PushNotificationActivity.java

```diff
@@ -64,7 +307,7 @@ public void onException(Exception e) {
                         PushNotificationActivity.this.runOnUiThread(new Runnable() {
                             @Override
                             public void run() {
-                                Toast.makeText(thisActivity, R.string.notification_error_network_failure_message, Toast.LENGTH_SHORT).show();
+                                Toast.makeText(thisActivity, R.string.notification_error_network_failure_message, Toast.LENGTH_LONG).show();
```

Comment:
Update the toast message to display the actual error - i.e. e.getLocalizedMessage()